cookbooks/bcpc/recipes/ssh.rb

```diff
@@ -74,118 +60,82 @@
   }
 end
 
-ssh_key_types = %w{dsa ecdsa ed25519 rsa}
-
-# First, try to retrieve existing host keys from the server.
-ruby_block 'get-ssh-secrets-from-server' do
-  block do
-    require 'chef-vault'
-    begin
-      node.run_state[:bcpc_ssh_host_keys] =
-        ChefVault::Item.load('ssh_host_keys', node[:fqdn])
-    rescue Exception => ee
-      # If we fail to load the vault, log the failure and create a vault.
-      Chef::Log.warn("Failed to load ssh_host_keys/#{node[:fqdn]}:\n#{ee}")
-
-      begin
-        vault_item = ChefVault::Item.new('ssh_host_keys', node[:fqdn])
-        vault_item.admins([get_bootstrap, node[:fqdn], 'admin'].join(','))
-        vault_item.search("fqdn:#{node[:fqdn]}")
-        vault_item.save
-        node.run_state[:bcpc_ssh_host_keys] = vault_item
-      rescue Exception => eee
-        # If we also fail to create, this node must not be an admin.
-        Chef::Log.warn('Failed to create new vault ' \
-                       "ssh_host_keys/#{node[:fqdn]}. " \
-                       "Is this node an admin?\n#{eee}")
-      end
-    end
-  end
-  not_if { node[:fqdn] == get_bootstrap }
+require 'chef-vault'
+
+def load_ssh_keys
```

Comment:
Please add a method comment on what you intend this function to do, return and consume.